ansible/roles/delfin-installer/scenarios/rabbitmq.yml

```diff
@@ -82,14 +82,11 @@
 
 - name: Install RabbitMQ package
   apt: 
-    name: "{{ item }}"
+    name: ['erlang-base', 'erlang-asn1', 'erlang-crypto', 'erlang-eldap', 'erlang-ftp', 'erlang-inets', 'erlang-mnesia', 'erlang-os-mon', 'erlang-parsetools', 'erlang-public-key', 'erlang-runtime-tools', 'erlang-snmp', 'erlang-ssl', 'erlang-syntax-tools', 'erlang-tftp', 'erlang-tools', 'erlang-xmerl', 'apt-transport-https', 'rabbitmq-server']
```

Comment:
Why do we need all these additional packages?

Reply:
@joseph-v As per [this](https://www.rabbitmq.com/install-debian.html#apt-quick-start-packagecloud) install guide we need to install these pacakges.